latex/Research_Strategy.tex

```diff
@@ -140,24 +140,24 @@
 Specifically, in some situations, free energy calculations appear to be capable of achieving RMS errors of 1-2 kcal/mol with current force fields, even in prospective applications, sufficient to drastically reduce the number of molecules that must be synthesized and assayed~\cite{shirts_free-energy_2010}.
 As a consequence, pharmaceutical companies are beginning to use these methods in active discovery projects.
 
 %Limitations of methods and of retrospective tests or application for methods improvements
-\textbf{Despite this progress, these methods currently have severe limitations} which prevent their widespread use in molecular design.
+\textbf{Despite progress, current physical modeling methodologies suffer from severe limitations hindering their widespread use in molecular design.}
 For example, even ``small'' protein conformational conformational changes not gracefully handled by current methodologies can yield errors up to 5 kcal/mol in calculated binding free energies~\cite{lim_sensitivity_2016}, force field limitations still pose major challenges~\cite{rocklin_blind_2013}, and the inability to treat important chemical effects like protonation and tautomer equilibria drastically limits the domain of applicability.
-For many pharmaceutically relevant systems, the most important sources of error are not yet clear.
+{\bf For many pharmaceutically relevant systems, the most important sources of error---and modeling challenges---are not yet clear.}
 
 Progress on addressing these challenges has been frustratingly slow, hindered by a lack of high-quality data and community focus.
-{\bf Neither retrospective tests nor prospective application in active drug discovery projects provides the necessary impetus and data to rapidly overcome the remaining major barriers to widespread utility.}
-Large-scale retrospective tests certainly assess our \emph{retrospective} performance, but they do not provide accurate guidance on utility for prospective design, nor do they effectively pave the way forward.
-One issue with retrospective tests is that they can easily result in over-fit models, or researchers applying a variety of different protocols until they get good results, perhaps even by a statistical fluke. %[ref] .
-%Probably should have some reference on the above.
+{\bf Neither retrospective tests nor prospective application in active discovery projects provides the necessary impetus and data to rapidly overcome remaining barriers to widespread utility.}
+Large-scale retrospective tests can assess \emph{retrospective} performance, but they do not provide accurate guidance on utility for prospective design, nor do they effectively pave the way forward.
+{\color{red}[JDC: Can we say something more concrete than ``effectively pave the way forward''?]}
```

Comment:
I'm open to suggestions. Has to be something super concise. "nor do they effectively identify the most important sources of error" or something like that would have been too long (at least in my PDF -- adds an extra line).